firefox-ios/Client/Frontend/Browser/Toolbars/SearchBarLocationSaver.swift

```diff
@@ -0,0 +1,36 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Shared
+
+class SearchBarLocationSaver: SearchBarLocationProvider, FeatureFlaggable {
```

Comment:
nit. Also I guess this needs to be a class and cannot be a struct to the protocol conformance?
```suggestion
final class SearchBarLocationSaver: SearchBarLocationProvider, FeatureFlaggable {
```